### DISCUSSION TITLE
python 3.13 Releases

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#1549] [Ideas]
https://www.python.org/downloads/

| library | 3.13 |
| ---- | ---- |
|apscheduler|?|
|aws-requests-auth|?|
|boto3|○|
|cffi|?|
|croniter|?|
|elasticsearch|?|
|envparse|?|
|exotel|?|
|Jinja2|○|
|jira|?|
|jsonpointer|?|
|jsonschema|○|
|prettytable|○|
|prison|?|
|prometheus_client|?|
|python-dateutil|?|
|PyYAML|?|
|py-zabbix|?|
|requests|?|
|sortedcontainers|?|
|statsd-tags|?|
|stomp.py|?|
|tencentcloud-sdk-python|?|
|texttable|?|
|twilio|?|

| library | 3.13 |
| ---- | ---- |
|pylint|○|
|pytest|○|
|pytest-cov|?|
|pytest-xdist|?|
|setuptools|?|
|sphinx|○|
|sphinx_rtd_theme|?|
|tox|?|

**Comment by nsano-rururu** [comment]
I changed to python 3.13 and no errors occurred when I ran make test-docker.

**Reply by jertel** [reply]
That's good news. If you want to submit a PR go ahead. The next ElastAlert 2 release won't go out for a couple months, so there's plenty of time for people to try the latest build and report issues.

**Reply by nsano-rururu** [reply]
Python 3.13 seems to require OpenSSL 3.0.8 or higher. The package installation documentation also needs to be updated. I recall it now states OpenSSL 1.1.1 or higher.

**Reply by nsano-rururu** [reply]
https://github.com/jertel/elastalert2/pull/1551
